### PR TITLE
Allow students to view each other's work for peer review on bubble choice levels

### PR DESCRIPTION
**Pull request by bencodeorg**
This change enables students to view each other's work on "bubble choice" levels, such as the "Asphalt Art Project" in the first unit of CSA, when peer review is enabled:

http://studio.code.org/s/csa1-pilot/lessons/16/levels/1

## Testing story

I tested manually that, while logged in as a student, I was able to view another student's work in my section who had enabled peer review on a bubble choice level. I also confirmed I was able as a teacher leading that section to provide review on my student's work on bubble choice levels (I don't think this was ever an issue though).

I also added a unit test to cover this case.

### CODE REVIEW

File: dashboard/app/controllers/script_levels_controller.rb [hunk 1]
```diff
@@ -423,12 +428,15 @@ def load_section
     end
   end
 
+  def select_bubble_choice_level
```
Comment (molly-moen) [inline dashboard/app/controllers/script_levels_controller.rb:431]: nice refactor :)

File: dashboard/test/models/ability_test.rb [hunk 2]
```diff
@@ -275,6 +275,26 @@ class AbilityTest < ActiveSupport::TestCase
     assert Ability.new(peer_reviewer).can? :view_as_user_for_code_review, javalab_script_level, project_owner
   end
 
+  test 'student in same CSA code review enabled section as student seeking code review can view as peer on bubble choice level' do
```
Comment (molly-moen) [inline dashboard/test/models/ability_test.rb:278]: nice test!

**Review by molly-moen — APPROVED** [review]
LGTM thanks for getting this done so quickly!

**Review by sanchitmalhotra126 — APPROVED** [review]
Nice fix!

**Pull request merged**